meteor/client/ui/Shelf/DashboardPanel.tsx

```diff
@@ -391,25 +391,42 @@ export function getUnfinishedPiecesReactive (rundownId: string, currentPartId: s
 			rundownId: rundownId,
 			partId: currentPartId,
 			dynamicallyInserted: true,
-			$or: [{
-				stoppedPlayback: {
-					$eq: 0
-				}
-			}, {
-				stoppedPlayback: {
-					$exists: false
-				}
-			}, {
-				definitelyEnded: {
-					$exists: false
+			$and: [
+				{
+					$or: [{
+						stoppedPlayback: {
+							$eq: 0
+						}
+					}, {
+						stoppedPlayback: {
+							$exists: false
+						}
+					}],
+				},
+				{
+					definitelyEnded: {
+						$exists: false
+					}
 				}
-			}],
+			],
 			playoutDuration: {
 				$exists: false
 			},
 			adLibSourceId: {
 				$exists: true
-			}
+			},
+			$or: [
+				{
+					userDuration: {
+						$exists: false
+					}
+				},
+				{
+					'userDuration.duration': {
+						$exists: false
+					}
+				}
+			]
```

Comment:
I think this should work? Double check my Boolean logic, but I think that's equivalent to the filter we had here before.

Reply:
Yeah, looks equivalent to me